client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -284,45 +360,61 @@ const CancelPurchaseRefundInformation = ( {
 		includedDomainPurchase &&
 		isDomainMapping( includedDomainPurchase )
 	) {
-		text = i18n.translate(
-			'This plan includes the custom domain mapping for %(mappedDomain)s. ' +
-				'The domain will not be removed along with the plan, to avoid any interruptions for your visitors. ',
-			{
-				args: {
-					mappedDomain: includedDomainPurchase.meta,
-					mappingCost: includedDomainPurchase.priceText,
-				},
-			}
+		text.push(
```

Comment:
I think that `text` is undefined here, so this call to `push()` will result in a fatal error.

Reply:
issue has been fixed

Reply:
Nice!